src/Traits/Body/ChecksForWithBody.php

```diff
@@ -19,10 +19,10 @@ trait ChecksForWithBody
      */
     public function bootChecksForWithBody(PendingRequest $pendingRequest): void
     {
-        if ($pendingRequest->getRequest() instanceof WithBody || $pendingRequest->getConnector() instanceof WithBody) {
+        if ($pendingRequest->getRequest() instanceof HasBody || $pendingRequest->getConnector() instanceof HasBody) {
             return;
         }
 
-        throw new BodyException(sprintf('You have added a body trait without adding the `%s` interface to your request/connector.', WithBody::class));
+        throw new BodyException(sprintf('You have added a body trait without adding the `%s` interface to your request/connector.', HasBody::class));
```

Comment:
I _think_ request/connector could get confused for the need to exist on both the request and connector.

```suggestion
        throw new BodyException(sprintf('You have added a body trait, without adding the `%s` interface to your request or connector.', HasBody::class));
```

I'm also a bit unsure on how this would work.
Would it be better to check
```php
if ($this instanceof HasBody) {
    return;
}
```

Otherwise we could technically add the trait to the connector, and the interface to the request(s), or vice versa.
I haven't checked if it is, but if it's supported to do that, then I think it should technically work, but might be a bit 'inconsistent', for the lack of a better word.